web-ui/package.json

```diff
@@ -48,13 +48,14 @@
     "universal-cookie": "^4.0.4"
   },
   "scripts": {
-    "start": "vite",
+    "build-storybook": "storybook build",
     "build": "vite build",
-    "serve": "vite preview",
-    "test": "vitest",
     "coverage": "vitest run --coverage --watch=false",
+    "format": "prettier --write 'src/**/*.{css,html,js,jsx}'",
```

Comment:
This is the only new script.

Reply:
Should we include `tsx` for futureproofing?

Reply:
I was thinking the same thing, and to include `.ts`, `.cjs` and possibly `.mjs` as well.

Reply:
I merged before I saw these comments. I will add those extensions in another PR.